container/root/scripts/install_s6.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash -e
+
+# Add S6 for zombie reaping, boot-time coordination, signal transformation/distribution
+# @see https://github.com/just-containers/s6-overlay
+#
+# Downloads, verifies, and extracts
+# Requires curl, gpg (or gnupg on Alpine), and tar to be present
+
+S6_NAME=s6-overlay-amd64.tar.gz
+S6_VERSION=v1.21.7.0
+
+curl -L https://github.com/just-containers/s6-overlay/releases/download/${S6_VERSION}/${S6_NAME} -o /tmp/${S6_NAME}
+curl -L https://github.com/just-containers/s6-overlay/releases/download/${S6_VERSION}/${S6_NAME}.sig -o /tmp/${S6_NAME}.sig
+curl https://keybase.io/justcontainers/key.asc| gpg --no-tty --batch --import
+gpg --no-tty --batch --verify /tmp/${S6_NAME}.sig /tmp/${S6_NAME}
```

Comment:
@bryanlatten : Does it make sense to chain these 4 commands e.g. curl -L && curl -L and so on?

I know in the BBC factory we've had situations where curl would succeed but it would be empty (or the contents would be 404). I suppose the gpg step would fail to verify but perhaps we can just fail if its not even able to download what it needs right off the bat.

Reply:
It can still be a little flakey but `curl --fail` might help return error codes on 4XX responses. 

In this case `curl -f -L` to keep the follow redirects behavior.

Reply:
@adobejmong @mannytoledo added the `-f` flag to fail more explicitly